readme.md

```diff
@@ -1,7 +1,7 @@
 # Moyo header
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://Hennadii-Turovskyi.github.io/layout_moyo-header/)
```

Comment:
![image](https://user-images.githubusercontent.com/38065883/174637337-759fcd68-3906-4d5f-9075-61b12dd69aab.png)
Box-shadow is missing

Reply:
![image](https://user-images.githubusercontent.com/38065883/174637441-1a36bd0b-6bba-402b-92f4-80715b6907d3.png)
There should be no margin-left on the first li item